scripts/pi-hole/php/footer.php

```diff
@@ -49,6 +49,7 @@
     $coreVersionStr = $core_current . (isset($core_commit) ? " (" . $core_branch . ", " . $core_commit . ")" : "");
     $ftlVersionStr = $FTL_current . (isset($FTL_commit) ? " (" . $FTL_branch . ", " . $FTL_commit . ")" : "");
     $webVersionStr = $web_current . (isset($web_commit) ? " (" . $web_branch . ", " . $web_commit . ")" : "");
+    $dockerTag = getenv('PIHOLE_DOCKER_TAG') == true ? getenv('PIHOLE_DOCKER_TAG') : "";
```

Comment:
On a second look, this seems odd. The function is [known to return](https://www.php.net/manual/en/function.getenv.php) either `string` or `false` but never `true` ? I guess we're using a side-effect of `== true` here?

```suggestion
    $dockerTag = getenv('PIHOLE_DOCKER_TAG');
```

Reply:
Cool, I just stackoverflowed the solution :) Let me test this...